guides/Git Shopify.md

```diff
@@ -0,0 +1,56 @@
+# Git strategy
+
+Our quick guide to working collaboratively with Git, GitHub and Shopify projects.
+
+## Create a Feature
+
+Create a `staging` branch from `master`.
+
+```
+git checkout master
+git pull
+git checkout -b staging
+```
+If the staging branch already exists, rebase it on-top of master:
+
+```
+git checkout staging
+git rebase master
+```
+
+Finally, push the staging branch to remote so we have a communal copy:
+```
+git push origin staging
+```
+
+## Production hotfix
+
+If you need to make a hotfix on production, create a new, local branch off `master`.
+
+```
+git checkout master
+git pull
+git checkout -b hotfix-some-bug
+```
+
+Once you have created fixed the bug, or added the small feature create a new Pull Request to `master`.
```

Comment:
Oops. This should read:

Once you have fixed the bug, or added the small feature, create a new Pull Request to `master`.